ocs_ci/utility/utils.py

```diff
@@ -4766,14 +4766,126 @@ def get_acm_version():
     from ocs_ci.ocs.resources.csv import get_csvs_start_with_prefix
 
     csv_list = get_csvs_start_with_prefix(
-        "advanced-cluster-management", namespace=constants.ACM_HUB_NAMESPACE
+        "advanced-cluster-management", namespace=namespace
     )
     for csv in csv_list:
         if "advanced-cluster-management" in csv["metadata"]["name"]:
             # extract version string
             return csv["spec"]["version"]
 
 
+def get_dr_hub_operator_version(namespace="openshift"):
+    """
+    Get DR Hub Opertaor Version
+
+    Returns:
+        str: returns version string
+    """
+    # Importing here to avoid circular dependency
+    from ocs_ci.ocs.resources.csv import get_csvs_start_with_prefix
+
+    csv_list = get_csvs_start_with_prefix(
+        constants.ACM_ODR_HUB_OPERATOR_RESOURCE, namespace=namespace
+    )
+    for csv in csv_list:
+        if constants.ACM_ODR_HUB_OPERATOR_RESOURCE in csv["metadata"]["name"]:
+            # extract version string
+            return csv["spec"]["version"]
+
+
+def get_ocp_dr_cluster_operator_version(namespace="openshift"):
```

Comment:
we will also need to add all of those values to test suite property via: record_testsuite_property

E.g. like we do record markers and other values:
https://github.com/red-hat-storage/ocs-ci/commit/088431523f5c4ad32775ed4fef2b047d4bfe0429

See other usages of record_testsuite_property in code via `git grep record_testsuite_property` please .

As main goal is have those values recorded in email, and report portal and xunit file as well.

`record_testsuite_property("rp_ocs_build", get_ocs_build_number())`

IIRC whatever we want to have in Report portal we need to prefix with `rp_` than our script will remove `rp_` prefix and report the attribute without this prefix.

Reply:
done